### ISSUE TITLE
it seems have a bug when I train a x2(scale=2) model?

### ISSUE THREAD

**Issue by xiaoxiongli** [NJU-Jet/SR_Mobile_Quantization#3]
Hi Dear @NJU-Jet:

I wanna to train a model that scale=2. (as you know, the defaut scale is 3)
it seems have a bug here:
![image](https://user-images.githubusercontent.com/13975044/120324237-fca16880-c318-11eb-9b23-0c7796cc04df.png)

you can see that this [inp, inp, inp, inp, inp, inp, inp, inp, inp] is for scale=3, so how to fix it if I want to train a x2 model?

can we fix it this way: [inp, inp, inp, inp, inp, inp, inp, inp, inp] --> **[inp]*scale*scale** ?

thank you very much!

**Comment by NJU-Jet** [comment]
You are right and this is a bug. The inp should just repeat (scale**2) times.
